libs/ledger-live-common/src/exchange/providers/swap.ts

```diff
@@ -141,7 +142,12 @@ export const getProvidersData = async () => {
   try {
     const providersData = await (
       await fetch(
-        "https://crypto-assets-service.api.aws.prd.ldg-tech.com/v1/partners?output=name,payload_signature_computed_format,signature,public_key,public_key_curve",
+        "https://crypto-assets-service.api.ledger.com/v1/partners?output=name,signature,public_key,public_key_curve&service_name=swap",
+        {
+          headers: {
+            "X-Ledger-Client-Version": `live-exchange/${PACKAGE.version}`,
```

Comment:
using `network` instead of `fetch` where network is from `live-network` will benefit the auto set of the Ledger-Client-Version header.

(moreover but comment that is beyond this PR: we should have defined a domain env in live-env, like all other services we use, maybe someone from live-hub can takes this over)

Reply:
Thanks, I switched it to network lib. Please double check @lpaquet-ledger, I am not familiar with this codebase